nuget/updater/common.ps1

```diff
@@ -86,8 +86,16 @@ function Install-Sdks([string]$jobFilePath, [string]$repoContentsPath, [string]$
 
     $sdksToInstall = Get-SdkVersionsToInstall -repoRoot $rootDir -updateDirectories $candidateDirectories -installedSdks $installedSdks
     foreach ($sdkVersion in $sdksToInstall) {
-        Write-Host "Installing SDK $sdkVersion"
-        & $dotnetInstallScriptPath --version $sdkVersion --install-dir $dotnetInstallDir
+        $versionParts = $sdkVersion.Split(".")
+        if ($versionParts.Length -eq 3 -and $versionParts[2] -eq "0") {
+            $channelVersion = "$($versionParts[0]).$($versionParts[1])"
+            Write-Host "Installind SDK from channel $channelVersion"
```

Comment:
nit: typo on installing

Reply:
Dangit!  I _just_ pushed a fix and hoped nobody saw it.